apps/picsa-apps/dashboard/src/app/modules/resources/resources.module.ts

```diff
@@ -22,6 +24,10 @@ import { ResourcesPageComponent } from './pages/home/resources.page';
         path: ':id',
         component: ResourceCreateComponent,
       },
+      {
+        path:'search',
+        // component:Sear/chComponent,
```

Comment:
nit(blocking)
It appears you have commented out the route you added, not sure if this is a typo or you plan to integrate in some other way, but please update as required

Reply:
Alright, i will make the necessary changes